### PR TITLE
feat(input-field): add support for a helper text next to the input field

### PR DESCRIPTION
**Pull request by jgroth**
### Browsers tested:

Windows:
- [ ] Chrome
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/input-field/input-field.tsx [hunk 1]
```diff
@@ -167,6 +173,9 @@ export class InputField {
                 {this.renderTrailingIcon()}
                 <div class="mdc-line-ripple" />
             </label>,
+            <div class="mdc-text-field-helper-line">
+                <div class="mdc-text-field-helper-text">{this.helperText}</div>
+            </div>,
```
Comment (adrianschmidt) [inline src/components/input-field/input-field.tsx]: I haven't tried it yet, so maybe MDC takes care of it, but I suspect this should only be output if there is actually any text in `this.helperText`?

Comment (adrianschmidt) [reply src/components/input-field/input-field.tsx]: ⚡️

File: src/components/input-field/input-field.tsx [hunk 2]
```diff
@@ -112,6 +112,7 @@ export class InputField {
     private action: EventEmitter<void>;
 
     constructor() {
+        this.renderHelperText = this.renderHelperText.bind(this);
```
Comment (jgroth) [inline src/components/input-field/input-field.tsx]: There is no need to bind this function, since it is called with `this.`

Comment (adrianschmidt) [reply src/components/input-field/input-field.tsx]: ⚡️

**Pull request merged**